env.template

```diff
@@ -1,7 +1,8 @@
-PLUGINS="example"
+PLUGINS="example google-analytics"
```

Comment:
I think we don't want this enabled by default since the user will need to add their GA ID anyway before it works.

Reply:
That makes sense.